### PR TITLE
RD-2960: standing order / scheduled payment adjustments + samples

### PR DESCRIPTION
**Pull request by gianluca-token**

### CODE REVIEW

**Comment by mahendra-aps** [comment]
@yuhanniu @gianluca-token 
We have also added StandingOrder and BulkTransfer latest changes as per Java Sdk to APS-REVAMP-01 
Are we looking to merge this branch as well, if yes can you please also review this pull request 
https://github.com/tokenio/sdk-csharp/pull/51

File: user/src/StandingOrderTokenBuilder.cs [hunk 1]
```diff
@@ -35,15 +35,15 @@ public sealed class StandingOrderTokenBuilder
         /// <param name="currency">currency of the token</param>
         /// <param name="frequency">ISO 20022 code for the frequency of the standing order:
         ///            DAIL, WEEK, TOWK, MNTH, TOMN, QUTR, SEMI, YEAR</param>
-        /// <param name="startDate">start date of the standing order: ISO 8601 YYYY-MM-DD or YYYYMMDD</param>
-        /// <param name="endDate">end date of the standing order: ISO 8601 YYYY-MM-DD or YYYYMMDD</param>
+        /// <param name="startDate">start date of the standing order
+        /// <param name="endDate">end date of the standing order
```
Comment (mahendra-aps) [inline user/src/StandingOrderTokenBuilder.cs]: closing tag </param> missing

File: tpp-sample/src/RedeemStandingOrderSampleTest.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿using Tokenio.Proto.Common.AliasProtos;
```
Comment (mahendra-aps) [inline tpp-sample/src/RedeemStandingOrderSampleTest.cs]: Name conflict in java and C# 
1. In java:
RedeemStandingOrderTokenSampleTest.java

in c#
RedeemStandingOrderSampleTest.cs

File: user-sample/src/CreateTransferTokenSampleTest.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                UserMember payer = TestUtil.CreateMemberAndLinkAccounts(tokenClient);
+                Alias payeeAlias = TestUtil.RandomAlias();
+                UserMember payee = tokenClient.CreateMemberBlocking(payeeAlias);
+                Token token = CreateTransferTokenSample.CreateTransferToken(payer, payeeAlias, Key.Types.Level.Low);
+                Assert.NotNull(token);
+            }
+        }
+
+        [Fact]
+        public void CreateScheduledPaymentTokenTest()
```
Comment (mahendra-aps) [inline user-sample/src/CreateTransferTokenSampleTest.cs]: Method Name conflict in java and C#.
 in java:
createPaymentTokenScheduledTest()

in c#
CreateScheduledPaymentTokenTest()

**Comment by gianluca-token** [comment]
@mahendra-aps thanks for the comments - I will put this on hold until you merge APS-REVAMP-01.

**Pull request closed without merging**